### PR TITLE
create license

### LINKED ISSUE #16: Add MIT LICENSE file to the repository.
# About issue-
Add MIT LICENSE file to the repository and add its link to the repo README.
Refer to the [documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository) for help.

`Copyright to Pranav Mendiratta and ALPHAVIO`

### PR DESCRIPTION
**Pull request by Swapna804**
Resolved  #16

### CODE REVIEW

**Comment by Pranav016** [comment]
Is this PR solving 2 issues?

**Comment by Pranav016** [comment]
Always follow one best practice, if you're resolving two issues, make separate branches for them and don't commit anything to the master.

**Comment by Swapna804** [comment]
Actually i am new to open source, will keep in mind, thankyou.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 BlogSite
```
Comment (Pranav016) [inline LICENSE:3]: You never write project name in copyright, always write org name.
In this case write 
`Pranav016 and ALPHAVIO`

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see
+https://www.contributor-covenant.org/faq
```
Comment (Pranav016) [inline CODE_OF_CONDUCT.md:76]: I requested some changes in these lines in #29 . Same mistakes were made. Please refer to it and correct them.

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Please make a new pull request with requested changes and from a new branch. Make separate PRs for separate issues.

**Pull request closed without merging**